docs/.vuepress/config.js

```diff
@@ -41,6 +41,46 @@ module.exports = {
         },
       },
     ],
+    ['@simonbrunel/vuepress-plugin-versions', {
+      menu: {
+        items: [
+          {
+            text: 'Documentation',
+            items: [
+              {
+                text: 'Development (master)',
+                link: '/docs/master/',
+              },
+              {
+                type: 'versions',
+                exclude: '^[0-2]\\.',
+                group: 'minor',
+                text: '{{version}} (latest)',
+                link: '/docs/{{version}}/'
+              },
+              {
+                type: 'versions',
+                exclude: '^[013]\\.|2\\.[0-5]\\.',
+                group: 'minor',
+                link: '/docs/{{version}}/'
+              }
+            ]
+          },
+          {
+            text: 'Release notes',
+            items: [
+              {
+                type: 'versions',
+                exclude: '^[01]\\.|2\\.[0-8]\\.',
+                target: '_blank',
+                group: 'patch',
```

Comment:
This one may be an issue at some point because it displays all versions. You can still adjust the `exclude` pattern but it would only apply to new deploys to the docs. Don't forget that already published docs will also get updated at runtime, so for example, after releasing 20 releases of v3.x.x, this list will contains more than 20 entries in published docs.

If you want to display just a few recent release notes, I can add a `limit` option to only return the `limit` most recent versions.

What do you think?

Reply:
I think a `limit` option would be good for this.

Reply:
Done: https://github.com/simonbrunel/vuepress-plugin-versions/commit/59856f3b35a07599a9724f384f95dc5912d0b658

Reply:
And applied